cmd/node/main.go

```diff
@@ -372,6 +373,13 @@ VERSION:
 		Name:  "import-db-no-sig-check",
 		Usage: "This flag, if set, will cause the signature checks on headers to be skipped. Can be used only if the import-db was previously set",
 	}
+
+	// redundancyLevel defines a flag that specifies the level of redundancy for the current instance (0 = master)
```

Comment:
To be "politically correct", let's replace `master` with `main` just about everywhere.

Reply:
done